index.js

```diff
@@ -15,9 +15,10 @@ const {
 const quickInterval = QUICK_NAP_DURATION_SECONDS * 1000,
   slowInterval = SLOW_NAP_DURATION_MINUTES * 60_000;
 
-async function main() {
-  console.log("Late No More Online..");
+const say= require("./src/applescript/say.js")
```

Comment:
nitpick: space on either side of =

Reply:
Have it fixed locally but VScode is not making this easy on me -- but it makes my eyes bleed too :)